integration/shareddata/shareddata.go

```diff
@@ -32,6 +32,12 @@ type Provider interface {
 	// Handlers returns handlers compatible with this provider.
 	Handlers() []string
 
+	// Validators returns validators for the given handler.
+	// For example, for Tigris it should return a map with they key $tigrisSchemaString
+	// and the value containing Tigris' JSON schema string.
+	// As Tigris schema requires collection name, please use %%collection%% as placeholder.
```

Comment:
This placeholder is a questionable thing for me, but the schema must have a collection name (and the names should match), so this approach looks to be the simplest.

Reply:
I think that `%%magic%%` is an implementation detail that is shared by a lot of code. I think it could be incapsulated

Reply:
Fixed